packages/cli/src/commands/runAndroid/__tests__/getLaunchPackageName.test.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import fs from 'fs';
+import getLaunchPackageName from '../getLaunchPackageName';
+
+jest.mock('fs');
+jest.mock('path');
+
+describe('run-android::getLaunchPackageName', () => {
+  const actualFs = jest.requireActual('fs');
+  const actualPath = jest.requireActual('path');
+
+  fs.readFileSync = jest.fn(filename => {
+    switch (filename) {
+      case 'app/build.gradle':
+        return actualFs.readFileSync(
+          actualPath.join(__dirname, '../__fixtures__/sampleBuild.gradle'),
+          'utf8',
+        );
+      // Use default case to catch generated debug manifest
+      default:
+        return actualFs.readFileSync(
+          actualPath.join(
+            __dirname,
+            '../__fixtures__/sampleGeneratedDebugManifest.xml',
+          ),
+          'utf8',
+        );
+    }
+  });
+
```

Comment:
please add a test case for flavor.buildType as well as just buildType

Reply:
Added a test for a flavored build. Let me know if it works.